modules/nw-ovn-kubernetes-live-migration.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+// * networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-live-migration_{context}"]
+= Migrating to the OVN-Kubernetes network plugin by using the live migration method
+
+The following procedure entails checking for egress router resources and using the live migration method to migrate from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin.
+
+.Prerequisites
+
+* A cluster has been configured with the OpenShift SDN CNI network plugin in the network policy isolation mode.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a recent backup of the etcd database is available.
+* The cluster is in a known good state without any errors.
+* On all cloud platforms after updating software, a security group rule must be in place to allow UDP packets on port `6081` for all nodes.
+* Cluster administrators must remove any egress router resources.
+* You have reviewed the "Considerations for live migration to the OVN-Kubernetes network plugin" section of this document.
+
+.Procedure 
+
+. On the {product-title} web console, click *Observe* -> *Metrics*.
+
+. In the *Expression* box, add the following queries, then click *Add query*:
++
+[source,text]
+----
+network_attachment_definition_instances{networks="egress-router"} > 0
+----
+
+. Click *Run queries*. If any data points are found, you should not proceed with the OpenShift SDN to OVN-Kubernetes live migration procedure. These custom resources must be removed before using the live migration procedure.
+
+. To remove an egress router, you can delete the egress router custom resource definition:
++
+[source,terminal]
+----
+$ oc delete crd egressrouters.network.operator.openshift.io
```

Comment:
do we need to remove resource egressrouters CRD?  I think remove the egress pod is enough

Reply:
If we only remove the CRD, wouldn't the pod be created again before we start the migration?

Reply:
I think that this is resolved with Step 1 cc @pliurh @zhaozhanqi